pkg/actions/nodegroup/create.go

```diff
@@ -33,12 +33,12 @@ type NodeGroup struct {
 	cfg             api.ClusterConfig
 	nodegroupFilter filter.NodeGroupFilter
 	clusterProvider eks.ClusterProvider
-	options         NodeGroupOptions
+	options         CreateOptions
 }
 
-// NewNodeGroup returns a NodeGroup create action
-func NewNodeGroup(
-	cfg api.ClusterConfig, nodegroupFilter filter.NodeGroupFilter, clusterProvider eks.ClusterProvider, options NodeGroupOptions,
+// NewCreateAction returns a NodeGroup create action
+func NewCreateAction(
+	cfg api.ClusterConfig, nodegroupFilter filter.NodeGroupFilter, clusterProvider eks.ClusterProvider, options CreateOptions,
```

Comment:
do we want to move this to use the `Manager` instead of a new `Nodegroup` type? (I know this was intended to be a small PR, but this makes the `nodegroup` package a tad bit more confusing)